workspaces/quay/plugins/quay/catalog-info.yaml

```diff
@@ -0,0 +1,55 @@
+# https://backstage.io/docs/features/software-catalog/descriptor-format#kind-component
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: backstage-community-quay
+  title: Quay plugin
+  description: Quay plugin for Backstage
+  annotations:
```

Comment:
Can you split this catalog-info.yaml and put the first one into workspaces/quay/ ? That was the idea behind having two catalog entries here, one for a workspace, one for the frontend-plugin itself.